doc/source/how-to/dns-configuration.rst

```diff
@@ -0,0 +1,91 @@
+DNS configuration
+=================
+
+As explained in :ref:`Documenting`, PyAnsys projects publish their documentation
+online under the following canonical name (CNAME) convention:
+
+``https://<product>.docs.pyansys.com``
+
+To request a CNAME for the ``pyansys.com`` domain, please contact
```

Comment:
```suggestion
To request a CNAME for the ``pyansys.com`` domain, contact
```
Avoid the use of "please" in technical documentation--according to all style guides!

Reply:
Thanks Kathy!!